fe/estimator.py

```diff
@@ -53,32 +65,57 @@ def simulate(lamb, box, x0, v0, final_potentials, integrator, equil_steps, prod_
         ctxt.add_observable(obs)
 
     prod_schedule = np.ones(prod_steps)*lamb
-    du_dl_freq = 5
-    full_du_dls = ctxt.multiple_steps(prod_schedule, du_dl_freq)
 
-    du_dl_mean, du_dl_std = np.mean(full_du_dls), np.std(full_du_dls)
+    # run MD, optionally pausing every du_dl_freq steps to extract x snapshot
+    if not get_trajectory:
```

Comment:
reminder to myself to just add a `x_t_freq` as well to multiple_steps later on so this manual chunking doesn't have to happen. It seems to be a common enough use case to get trajectories.